bundles/org.openhab.binding.freebox/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -545,12 +564,20 @@
 		<item-type>String</item-type>
 		<label>Play Audio or Video URL</label>
 		<description>Play an audio or video media from the given URL</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>MediaControl</tag>
```

Comment:
"Status" is not appropriate, this channel does control playback.
I am a little lost with semantic tags: MediaControl" is a point or a property ?

Reply:
Mediacontrol is a property. It is used with Status if it is showing the playing state of a media player. Or with Control if OH can command the player.

Reply:
@lolodomo I changed it to Control+MediaPlayer. Please comment if you do not agree.

Reply:
Agree